docs/websockets.rst

```diff
@@ -245,4 +245,24 @@ Finally the backend provides more granular information to know what interval of
 - ``event_type``: Labels the type of events being queried. Valid values are: ``history_query``.
 - ``location``(Optional): When the same ``event_type`` can be queried in multiple locations this helps to differentiate them.
 - ``name``(Optional): If multiple appearences of the same location are possible it will differentiate each one of them.
-- ``period``: Time range that is being queried.
\ No newline at end of file
+- ``period``: Time range that is being queried.
+
+
+A need in balances refresh 
+============================
+
+If at some point backend detects that balances need to be refreshed, it will send this message to the frontend.
+
+::
+
+    {
+        "type": "refresh_balances",
+        "data": {
+            "section": "blockchain_balances",
+            "blockchain": "optimism"
+        }
+    }
+
+
+- ``section``: Balances section that needs a refresh.
```

Comment:
list the possible values for this field. Currently one but better keep a list in the docs